### ISSUE TITLE
Firefox extensions popup: Unable to focus/click on items on popup window

### ISSUE THREAD

**Issue by tskaar** [mahkoh/jay#329]
Hi,

When opening any of the pop-up windows from any extension on Firefox I'm unable to focus on (most of, top section seems to be fine?) the window, where Jay only focuses on whatever is underneath the window pop-out. Hovering over a button for multiple seconds allows the focus to change to the extension button, however it still clicks on the item underneath the window.

An example here with uBlock Origin on bloomberg.com:
![image](https://github.com/user-attachments/assets/1aa4f2e3-61ef-468d-b9ab-d3ae4b04a3ad)
Where the tooltip popup is from the article underneath the uBlock window

Let me know if you want/need me to test different settings to be able to reproduce this
Thanks!

# Environment / Settings

## HW
```text
iGPU: Radeon 680M
dGPU: Radeon RX 7600S
```

## Jay
```text
GfxApi: Vulkan,
iGPU DirectScanout: true,
dGPU DirectScanout: true,
PrimaryRender: dGPU
```
## SW
```text
Firefox: v133.0
Jay: main
```
## Env

```text
MOZ_ENABLE_WAYLAND=1
```

**Comment by mahkoh** [comment]
I'll look at it over the weekend.

**Comment by mahkoh** [comment]
Should be fixed by #331.

**Comment by tskaar** [comment]
Tested #331, working as expected now

Thanks for all the work!
